src/coreclr/jit/codegen.h

```diff
@@ -1477,7 +1477,7 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 public:
     void instGen(instruction ins);
 
-    void inst_JMP(emitJumpKind jmp, BasicBlock* tgtBlock);
+    void inst_JMP(emitJumpKind jmp, BasicBlock* tgtBlock, bool isJmpAlways = false);
```

Comment:
The name `isJmpAlways` is very confusing: we are always generating an unconditional branch by calling this function, so when would `isJmpAlways` be false?

Should this be named something like `canJmpToNextInstBeRemoved`? Or simpler, `isJmpRequired`? (with a comment defining what "required" means)